recipe/meta.yaml

```diff
@@ -22,30 +22,35 @@ requirements:
     - cross-python_{{ target_platform }}     # [build_platform != target_platform]
     - cython                                 # [build_platform != target_platform]
     - numpy                                  # [build_platform != target_platform]
+    - meson-python                           # [build_platform != target_platform]
+    - ninja                                  # [build_platform != target_platform]
+    - pkg-config                             # [unix]
     # pythran code needs clang-cl on windows
     - clang                                  # [win]
     - {{ compiler("c") }}
     - {{ compiler("cxx") }}
   host:
     - python
+    - python-build
     - pip
-    - cython >=0.29.21,!=0.29.18
+    - libclang-cpp                           # [win]
```

Comment:
For clang-cl.exe to be detected.

Reply:
Code generation through pythran

Reply:
Why does that need `libclang-cpp.dll`?

Reply:
It doesn't. I just don't know exactly which of the clang packages to include.....

Reply:
This was the first i tried

Reply:
clang.exe not found when only clang is installed

Reply:
I think I understand, the goal is to compile with clang instead of msvc in order to ensure that we can use pythran.

The failure is coming from the `bld.bat` script. What executable should we set for `CC` and `CXX` on windows:
https://github.com/hmaarrfk/scikit-image-feedstock/blob/0.20.0_update/recipe/bld.bat#L11

Reply:
We build our wheels on github actions with `clang.exe` as far as I know.

Reply:
Should be fixed with https://github.com/conda-forge/clangdev-feedstock/pull/207

Reply:
I don't see the relationship between https://github.com/conda-forge/clangdev-feedstock/pull/207 and not detecting clang...